Source/Python/WarpX.cpp

```diff
@@ -146,6 +147,29 @@ void init_WarpX (py::module& m)
             },
             py::arg("potential")
         )
+
+        .def("set_plasma_lens_strength",
+            [](WarpX& wx, int i_lens, amrex::Real strength_E, amrex::Real strength_B) {
+                auto & mypc = wx.GetPartContainer();
```

Comment:
Since this is affecting only `mypc`, should this go into `Source/Python/Particles/MultiParticleContainer.cpp` instead?

Reply:
Good point!  cc @RemiLehe fyi, I will move it

Reply:
Done! :)